src/components/ExperimentalDynamicWidgets.js

```diff
@@ -0,0 +1,10 @@
+import AisDynamicWidgets from './DynamicWidgets';
+import { warn } from '../util/warn';
+
+// @MAJOR remove this file
+export default Object.assign({}, AisDynamicWidgets, {
```

Comment:
Why `Object.assign` vs. spread?

Reply:
Vue InstantSearch babel wasn't set up to have spread